packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -294,13 +292,16 @@ const PlanStep: React.FunctionComponent< PlanStepProps > = ( {
 		select( LAUNCH_STORE ).getPaidPlanProductId()
 	);
 
-	const selectedPlan = useSelect( ( select ) =>
-		select( PLANS_STORE ).getPlanByProductId( selectedPlanProductId, locale )
-	);
+	const { selectedPlan, selectedPaidPlan, selectedPlanBillingPeriod } = useSelect( ( select ) => {
+		const plansStore = select( PLANS_STORE );
 
-	const selectedPaidPlan = useSelect( ( select ) =>
-		select( PLANS_STORE ).getPlanByProductId( selectedPaidPlanProductId, locale )
-	);
+		return {
+			selectedPlan: plansStore.getPlanByProductId( selectedPlanProductId, locale ),
+			selectedPaidPlan: plansStore.getPlanByProductId( selectedPaidPlanProductId, locale ),
+			selectedPlanBillingPeriod: plansStore.getPlanProductById( selectedPlanProductId )
```

Comment:
As explained in [this comment](https://github.com/Automattic/wp-calypso/pull/49412#pullrequestreview-582249246), when there isn't a selected plan in the launch store, `selectedPlanBillingPeriod` is `undefined`.

We should make sure that selectedPlanBillingPeriod has a default, fallback value of `ANNUALLY`